hutch_python/load_conf.py

```diff
@@ -11,6 +11,7 @@
 from bluesky.callbacks.best_effort import BestEffortCallback
 from bluesky.utils import install_kicker
 from pcdsdevices.mv_interface import setup_preset_paths
+from elog import HutchELog
```

Comment:
Ok, this is even more dumb than the other things, but can you put this into the correct alphabetical spot? It's going to bug me forever having one thing out of order.